engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetBlend.java

```diff
@@ -15,48 +15,55 @@
  */
 package org.terasology.rendering.dag.stateChanges;
 
+import static org.lwjgl.opengl.GL11.GL_BLEND;
 import org.terasology.rendering.dag.RenderPipelineTask;
 import org.terasology.rendering.dag.StateChange;
+import org.terasology.rendering.dag.tasks.DisableCapabilityTask;
 import org.terasology.rendering.dag.tasks.EnableCapabilityTask;
 
 /**
  * TODO: Add javadocs
  */
-public class EnableCapability implements StateChange {
-    private int capability;
-    private EnableCapabilityTask task;
+public final class SetBlend extends SetCapability {
```

Comment:
The term "capability" conflicts with the OpenGL concept with the same name, detailing the capabilities of a particular opengl implementation. For this reason I'd recommend to extend a class named SetStateParameter instead, to remove ambiguities.